cpp/open3d/geometry/PointCloudPlanarPatchDetection.cpp

```diff
@@ -62,7 +62,7 @@ class BoundaryVolumeHierarchy {
                             const Eigen::Vector3d& min_bound,
                             const Eigen::Vector3d& max_bound,
                             size_t min_points = 1,
-                            double min_size = 0.0)
+                            double min_size = 1e-10)
```

Comment:
Hi @nicolaloi thanks for submitting this! Looks good. Please add a comment here explaining the reason for this value.

Reply:
This is just a “drive by” comment from someone unfamiliar with this code. If the idea is to use any small, nonzero value, perhaps instead of a mysterious constant, could` std::numeric_limits<double>::epsilon()` be used instead? Its value appears to be `2.22045e-16`

Reply:
Yes, I'll add a comment and I can change it to `std::numeric_limits<double>::epsilon()`. It will still be an "arbitrary" value since `epsilon()` will be used as a "normal" small-enough threshold, and not as its intended specific difference between 1.0 and its next float value. However, indeed, it better expresses the idea of having a general small-enough threshold compared to a hardcoded number.

Reply:
I think you want the next highest representable value after 0.0: epsilon is the right number. If you actually want the next highest value after 1.0, it's`std::nextafter(1.0, INFINITY)`

Reply:
I'll add `std::numeric_limits<double>::epsilon()`, but it is not the next highest representable value after 0.0: [https://en.cppreference.com/w/cpp/types/numeric_limits/epsilon](https://en.cppreference.com/w/cpp/types/numeric_limits/epsilon). `std::numeric_limits<double>::epsilon()` gives 2.22045e-16, while `std::nextafter(0.0, INFINITY)` gives 4.94066e-324. But 4.94066e-324 I think is a too small number, which could give problems (too-deep recursion).

Reply:
Yes, I was initially going to suggest `std::numeric_limits<double>::min()` (which is 2.22507e-308) but wondered if it might be “too-small” as @nicolaloi said.